### ISSUE TITLE
Allow alternative dictionary directory

### ISSUE THREAD

**Issue by CsatiZoltan** [ismail/mobidict#2]
If the dictionaries are not in a specific directory, the program does not start.

**Comment by ismail** [comment]
Is this really needed? The dictionaries are really specific and cannot be shared with other programs. What kind of use case do you have in mind?

**Comment by CsatiZoltan** [comment]
> The dictionaries are really specific and cannot be shared with other programs.

True, I was just thinking of giving the user the choice to store their dictionaries in the directory they want (e.g. if their existing directory is already synced with the cloud).

**Comment by ismail** [comment]
> True, I was just thinking of giving the user the choice to store their dictionaries in the directory they want (e.g. if their existing directory is already synced with the cloud).

Ok, that's a valid use case, I'll add an option in the next release.
